mllib/src/test/scala/org/apache/spark/ml/regression/LinearRegressionSuite.scala

```diff
@@ -59,6 +60,15 @@ class LinearRegressionSuite extends SparkFunSuite with MLlibTestSparkContext {
     datasetWithoutIntercept = sqlContext.createDataFrame(
       sc.parallelize(LinearDataGenerator.generateLinearInput(
         0.0, Array(4.7, 7.2), Array(0.9, -1.3), Array(0.7, 1.2), 10000, seed, 0.1), 2))
+
+    val r = new Random(seed)
+    val featureSize = 4100
+    datasetWithBigFeature = sqlContext.createDataFrame(
+      sc.parallelize(LinearDataGenerator.generateLinearInput(
+        0.0, Seq.fill(featureSize)(r.nextDouble).toArray,
```

Comment:
It would be nice to use keyword arguments, it is hard to guess what `0.0, `200`, and `0.1` mean.